lib/t_web/channels/feed_channel.ex

```diff
@@ -32,11 +34,18 @@ defmodule TWeb.FeedChannel do
         |> Matches.list_matches()
         |> render_matches(screen_width)
 
+      expired_matches =
+        user_id
+        |> Matches.list_expired_matches()
+        |> render_expired_matches(screen_width)
+
       reply =
         %{}
+        |> Map.put("match_expiration_duration", @match_expiration_duration)
```

Comment:
I'd call it [`match_ttl`](https://en.wikipedia.org/wiki/Time_to_live), at least that's how resources with limited life-span are usually called.